rtloader/common/builtins/aggregator.c

```diff
@@ -189,16 +189,17 @@ static PyObject *submit_metric(PyObject *self, PyObject *args)
     char **tags = NULL;
     int mt;
     double value;
+    bool flush_first_value;
```

Comment:
Should it be `bool flush_first_value = false` as `flush_first_value` is an optional argument?